### PR TITLE
examples/bluetooth/MIT-App-Inventor: Add MIT App Inventor bluetooth example

### PR DESCRIPTION
**Pull request by vlmarek**

### CODE REVIEW

**Comment by vlmarek** [comment]
This example is supposed to give easy starting point when integrating MicroPython bluetooth code with MIT App Inventor project.

File: examples/bluetooth/MIT-App-Inventor/ble.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# This code requires the micropython_dotstar and ble_advertising libraries
```
Comment (stinos) [inline examples/bluetooth/MIT-App-Inventor/ble.py:1]: This is obvious from the imports, not really needed to repeat?

**Comment by stinos** [comment]
Are the .png and .zip really needed? The code alone should be sufficient, no?

File: examples/bluetooth/MIT-App-Inventor/ble.py (hunk 1 above)
Comment (vlmarek) [reply examples/bluetooth/MIT-App-Inventor/ble.py:1]: Thank you for looking into this request. I'm happy to remove that should you find this example worth including.

**Comment by vlmarek** [comment]
Well the .zip contains the MIT App Inventor sources. And the png files are screenshots of the project once you import it into your workspace. I find both helpful, you can take a look what is going on there and you can import whole project. So the .png and .zip are in a sense redundant, yes. I would prefer to have both, but you tell me.

Thank you
__ 
  Vlad

**Comment by stinos** [comment]
It's just that I doubt this will ever get merged when it contains seemingly random extra binary files which aren't required to run. If anything you could add a comment in the .py file with a link to the sources or so, not sure.

**Comment by vlmarek** [comment]
If the problem is the binary data, I can't do much about it. The example is supposed to show how to connect "A" and "B" and contains sources for "A" (== *.py) and "B" (== *.zip). If the problem is the "seemingness" of unrelated data, how about some explanation in README, maybe some link to App Inventor or such?

And it's totally ok to not want to have the code in the gate, no hard feelings :)
Cheers
__ 
  Vlad

**Comment by jimmo** [comment]
Thanks @vlmarek 

I agree with what @stinos is saying about the app inventor part, but at the same time this is still a really useful minimal example of how to write a GATT server (perhaps more useful as a starting point than the ble_temperature.py that is currently there which does too many things).

The fact that it's able to be controlled via App Inventor is interesting, but not really specifically relevant to the MicroPython code (i.e. you could equivalently control this from nRF Connect or anything -- the important thing is that it's showing how to make a simple writeable GATT characteristic).

Perhaps, like @stinos says, you could post the screenshots and app inventor code as a GitHub gist, and link to them from the example (along with maybe instructions of how you could control this via nRF Connect, or possibly even from MicroPython).

**Comment by dpgeorge** [comment]
It would be good to have a very simple GATT server example.  But in its current form this PR is not suitable with the additional images.

**Pull request closed without merging**